docs/blog/2021-06-21-zephyr-2-5.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Zephyr 2.5 Update"
+author: Pete Johanson
+author_title: Project Creator
+author_url: https://gitlab.com/petejohanson
+author_image_url: https://www.gravatar.com/avatar/2001ceff7e9dc753cf96fcb2e6f41110
+tags: [firmware, zephyr, core]
+---
+
+Happy to announce that we have completed the work to upgrade ZMK to [Zephyr 2.5](https://docs.zephyrproject.org/2.5.0/releases/release-notes-2.5.html)!
+
+A big part of this work was some _major_ refactors and improvements by [innovaker] to our [zmk-docker](https://github.com/zmkfirmware/zmk-docker/) Docker image and GH Actions automation. With the updated changes there, we can now:
```

Comment:
I'd drop the lead-in sentence to provide more flexibility for the grammar used in the bullet points ...
```suggestion
A big part of this work was some _major_ refactors and improvements by [innovaker] to our [zmk-docker](https://github.com/zmkfirmware/zmk-docker/) Docker image and GH Actions automation.
```